Something wrong with conditionals:
```
asciidoctor: ERROR: master.adoc: line 115: unmatched preprocessor directive: endif::[]
asciidoctor: ERROR: master.adoc: line 149: detected unterminated preprocessor conditional directive: ifndef::containerized[]
CHECKING FOR ASCIIDOCTOR ERRORS AND WARNINGS
make[2]: *** [../common/Makefile:86: ../build/Administering_Project/index-containerized-katello.html] Error 1
```

--- a/guides/common/modules/ref_starting-and-stopping-project-services.adoc
+++ b/guides/common/modules/ref_starting-and-stopping-project-services.adoc
@@ -33,7 +33,7 @@ To start {Project} services, execute:
 
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# systemctl service start foreman.target
+# systemctl start foreman.target
 ----
 
 To restart {Project} services, execute:

--- a/guides/common/modules/ref_starting-and-stopping-project-services.adoc
+++ b/guides/common/modules/ref_starting-and-stopping-project-services.adoc
@@ -9,6 +9,43 @@
 After installing {Project} with the `{foreman-installer}` command, all {Project} services are started and enabled automatically.
 View the list of these services by executing:
 
+ifdef::containerized[]
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# systemctl list-dependencies foreman.target
+----
+
+To see the status of running services, execute:
+
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# systemctl service status foreman.target
+----
+
+To stop {Project} services, execute:
+
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# systemctl service stop foreman.target
+----
+
+To start {Project} services, execute:
+
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# systemctl service start foreman.target
+----
+
+To restart {Project} services, execute:
+
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# systemctl service restart foreman.target
+----
+
+endif::[]
+
+ifndef::containerized[]
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
 # {foreman-maintain} service list
@@ -41,3 +78,4 @@ To restart {Project} services, execute:
 ----
 # {foreman-maintain} service restart
 ----
+endif::[]

--- a/guides/common/modules/ref_starting-and-stopping-project-services.adoc
+++ b/guides/common/modules/ref_starting-and-stopping-project-services.adoc
@@ -19,7 +19,7 @@ To see the status of running services, execute:
 
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# systemctl status foreman.target
+# systemctl list-units $(systemctl show -p Wants --value foreman.target)
 ----
 
 To stop {Project} services, execute:

